src/lib/components/FieldSelect.svelte

```diff
@@ -151,4 +196,12 @@
 	:global(.field-combobox-item-label) {
 		@apply w-full;
 	}
```

Comment:
Here are some styles for ya:

```suggestion
	:global(.field-combobox-item-label) {
		@apply w-full grid grid-cols-[auto,max-content];
	}
```

![image](https://github.com/user-attachments/assets/b8bf23bc-02c9-44d2-84d2-9301ca635e44)

Reply:
I made it look like this at the beginning because it looks good on the New session screen, but in the Settings screen I'm not sure if it looks nice because the input is wider. What do you think?

<img width="844" alt="image" src="https://github.com/user-attachments/assets/c91b8a79-1476-4e98-b1c8-86cd2df04572">